mk/re.mk

```diff
@@ -48,6 +48,10 @@
 #   PKG_CONFIG     Defined if pkg-config available
 #
 
+# Verbose and silent build modes
+ifeq ($(V),)
+HIDE=@
+endif
```

Comment:
This has to be placed in Makefile, other apps include `re.mk` and have own HIDE definitions.